src/components/icon/icons/feed.jsx

```diff
@@ -0,0 +1,53 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import Icon from '../icon';
+
+export default function Feed({
+  stroke,
+  fill,
+  strokeWidth,
+  style: styleProp,
+  ...rest // eslint-disable-line comma-dangle
+}) {
+  const style = {
+    ...styleProp,
+    ...Icon.defaultProps.style,
+  };
+  return (
+    <svg style={style} {...rest}>
+      <g stroke="none" strokeWidth={strokeWidth} fill={fill} fillRule="evenodd">
+        <g fill={fill}>
```

Comment:
You should be able to remove this `<g>` since there is already a `fill` on its parent